src/sentry/incidents/endpoints/serializers.py

```diff
@@ -151,189 +348,110 @@ def update(self, instance, validated_data):
+        """
+        print("running validate.!?", data)
+        triggers = data.get("triggers", [])
+        if triggers:
+            if len(triggers) == 1:
+                if triggers[0].get("label", None) != CRITICAL_TRIGGER:
+                    raise serializers.ValidationError("First trigger must be labeled 'critical'")
+            elif len(triggers) == 2:
+                if triggers[0].get("label", None) != CRITICAL_TRIGGER or triggers[1]["label"] != WARNING_TRIGGER:
+                    raise serializers.ValidationError("First trigger must be labeled 'critical', second trigger must be labeled 'warning'")
+                else:
+                    if triggers[0]["thresholdType"] != triggers[1]["thresholdType"]:
+                        raise serializers.ValidationError("Must have matching threshold types (i.e. critical and warning triggers must both be an upper or lower bound)")
+
+                    if triggers[0]["thresholdType"] == 0:
+                        if triggers[0]["alertThreshold"] < triggers[1]["alertThreshold"]:
+                            raise serializers.ValidationError("CRITICAL trigger must have an alert threshold above WARNING trigger")
+                        elif triggers[0]["resolveThreshold"] > triggers[1]["resolveThreshold"]:
+                            raise serializers.ValidationError("CRITICAL trigger must have a resolution threshold below (or equal to) WARNING trigger")
+                    elif triggers[0]["thresholdType"] == 1:
+                        if triggers[0]["alertThreshold"] > triggers[1]["alertThreshold"]:
+                            raise serializers.ValidationError("CRITICAL trigger must have an alert threshold below WARNING trigger",)
+                        elif triggers[0]["resolveThreshold"] < triggers[1]["resolveThreshold"]:
+                            raise serializers.ValidationError("CRITICAL trigger must have a resolution threshold above (or equal to) WARNING trigger")
+                    else:
+                        raise serializers.ValidationError("Invalid thresholdType supplied - must be 0 or 1")
+            else:
+                raise serializers.ValidationError("Must send 1 or 2 triggers - either 1 CRITICAL trigger, or 1 CRITICAL and 1 WARNING trigger")
+
+            # Triggers have passed checks. Check that all triggers have at least one action now.
+            for trigger in triggers:
+                actions = trigger.get("actions", [])
+                if actions == []:
+                    raise serializers.ValidationError(trigger["label"] + " trigger must have an action.")
+        else:
+            raise serializers.ValidationError("Must include at least one trigger")
+
+        # return self._handle_old_fields_transition(data)
+        print("validated so far, gonna return ",super(UnifiedAlertRuleSerializer, self)._handle_old_fields_transition(data))
```

Comment:
Probably don't need this anymore. If we're still passing multiple aggregations we can fix it on the FE as part of these changes.